### PR TITLE
merged MetricClock/MetricDatabase into IntervalAggregator

### PR DESCRIPTION
**Pull request by nsauro**

### CODE REVIEW

File: colossus-metrics/src/main/scala/colossus/metrics/MetricClock.scala [hunk 1]
```diff
@@ -17,40 +17,50 @@ class MetricDatabase(systemId: MetricSystemId, namespace: MetricAddress, snapsho
...
-  var build: MetricMap = blankMap() 
+  var build: MetricMap = blankMap()
   var metrics = systemMetrics.metrics
   val collectors = new JHashSet[ActorRef]()
+  var latestTick = 0L
 
-  //TODO: this name needs to be able to support multiple databases/metric systems.
-  val collectedGauge = new ConcreteGauge(Gauge(namespace / "metric_completion"), self)
+  val metricSafeDurationName = interval.toString().replaceAll(" ", "") //ie: 1second, 44milliseconds
+  val collectedGauge = new ConcreteGauge(Gauge(namespace / metricSafeDurationName / "metric_completion"), self)
```
Comment (DanSimon) [inline colossus-metrics/src/main/scala/colossus/metrics/MetricClock.scala]: FYI my merge broke this line I think, ConcreteGauge doesn't take the actor as a param anymore.

**Comment by DanSimon** [comment]
Just rename the file MetricClock to IntervalAggregator and I think this is good to go

**Comment by nsauro** [comment]
rebased the branch...fixed 2 merge conflicts

**Comment by DanSimon** [comment]
:+1:

**Pull request merged**